test/CheckedC/generic-function/generic-func-call-info.c

```diff
@@ -0,0 +1,67 @@
+// Test to make sure that the generic function calls are correctly instantiated
+//
+// In CallExpr, the type of the DeclRefExpr should be completely instantiated,
+// by replacing all type variable types with specified type names in the list
+// of type names provided when calling generic function.
+//
+//
+// RUN: %clang_cc1 -ast-dump -fcheckedc-extension %s | FileCheck %s
+
+
+_For_any(T, S, R) T genericTest(T test, S steve, R ray) {

```

Comment:
This test won't work after you make type variables be incomplete types.  Variables cannot have an incomplete type.